database/buildSchema/33_file.sql

```diff
@@ -12,7 +12,6 @@ CREATE TABLE public.file (
     is_output_doc boolean DEFAULT FALSE NOT NULL,
     is_internal_reference_doc boolean DEFAULT FALSE NOT NULL,
     is_external_reference_doc boolean DEFAULT FALSE NOT NULL,
-    is_missing boolean DEFAULT FALSE NOT NULL,
```

Comment:
We don't need this any more -- I found a better way to process the missing records. I didn't go with the Cursor, as that would need to install a whole extra package [node-pg-cursor](https://www.npmjs.com/package/pg-cursor), but I achieved something similar using GraphQL query with pagination to limit the batch size.